web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -383,6 +414,37 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
     });
   };
 
+  const handleLaunchDateChange = (val, period) => {
+    const isoDate = val?.$d.toISOString() ?? null;
+    const newPeriod = {
+      ...period,
+      launchDate: isoDate
+    };
+
+    setPeriodToAdd(newPeriod);
```

Comment:
If you move line 424 into the `updateReviewPeriodDates` function then you can delete lines 424, 434, and 444.

Reply:
Good idea.